showcase/src/components/ui/graph.tsx

```
@@ -15,147 +15,153 @@ const graphVariants = cva(
+  ) => {
     // Transform data for Recharts
     const chartData = data.labels.map((label, index) => ({
       name: label,
-      ...data.datasets.reduce((acc, dataset) => ({
-        ...acc,
-        [dataset.label]: dataset.data[index]
-      }), {})
-    }))
+      ...data.datasets.reduce(
+        (acc, dataset) => ({
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)